src/backends/perf_pt/mod.rs

```diff
@@ -785,6 +785,15 @@ mod tests {
         test_helpers::test_ten_times_as_many_blocks(tracer1, tracer2);
     }
 
+    // Check that our block decoder agrees ptxed on long trace.
+    // XXX We use an even higher iteration count once our decoder uses a libipt image cache.
+    #[ignore] // Decoding long traces is slow.
```

Comment:
How slow is slow?

Reply:
This test takes about a minute. It's long enough to annoy me during development.

Reply:
OK.